source/lex.tex

```diff
@@ -728,6 +728,7 @@
 \tcode{static}            &
 \tcode{unsigned}          \\
 
+\tcode{char8_t}
```

Comment:
No & ?

Reply:
No. See the "notes" above:
"P1073R3 Immediate functions (pull request #2461) contains a reorganization of the keyword table in [lex.key] as a \multicolfloattable, making further updates easier. It is recommended to apply the other pull request first and then rebase this one."